### DISCUSSION TITLE
Additional parameters for the prices/current endpoint

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#40] [Announcements]
I've added a `previous` and `next` param to the `prices/current` endpoint. You can specify how many intervals you would like. This will make it super easy to fetch rolling windows of actuals and forecasts.

So to get the current price plus three forecasts, you can do:

```
curl -X 'GET' \
  'https://api.amber.com.au/v1/sites/000000000000000000000/prices/current?next=3&previous=0' -H 'Authorization: Bearer psk_xxx'
```

[Docs have been updated](https://api.amber.com.au/developers).

**Comment by craigcurtin-dev** [comment]
You are a good man Charlie Brown - thats great work thanks

Craig

**Comment by edwapj** [comment]
Super easy indeed !
Thanks very much  👍

**Comment by oldwindmill** [comment]
excellent...

**Comment by MikeatLorne** [comment]
Really good,   I tried it for 5 periods and it worked fine.  Might be worth mentioning this in the instruction?

**Reply by madpilot** [reply]
Awesome!

Which instructions are you referring to? It's already in
The docs.

**Reply by MikeatLorne** [reply]
sorry - did not see it there - all good
